app/views/reviews/show.html.erb

```diff
@@ -20,12 +20,25 @@
 
 <h3>Ratings</h3>
 
-<p>Was this review kind, specific, and actionable?</p><br />
+<p id='rating-criteria'>Was this review kind, specific, and actionable?</p><br />
+
+<div id='new-rating' class='load-form'></div>
 
 <div>
   <div class='mini-box' id='new-rating-box'>
     <div class='squarebrackets'>
-      <center><%= link_to "+ rate review", new_rating_path(@review), class: 'body-link new-form-link', remote: true, id: 'new-rating-link' %></center>
+      <center>
+        <div class='body-link'>
+          <p>+ rate review</p><br />
+          <%# <%= link_to "+ rate review", new_rating_path(@review), class: 'new-form-link', remote: true, id: 'new-rating-link' %>
+          <%= link_to new_rating_path(@review), class: 'new-form-link', remote: true, id: 'new-rating-up'  do %>
+            <i class="fa fa-thumbs-up fa-2x" aria-hidden="true"></i>
+          <% end %>
+          <%= link_to new_rating_path(@review), class: 'new-form-link', remote: true, id: 'new-rating-down'  do %>
+            <i class="fa fa-thumbs-down fa-2x" aria-hidden="true"></i>
+          <% end %>
```

Comment:
Not sure why these are not taking the hover style of the div's body-link class. Can you add this to the `main.css` to make the hover work?

```

a.new-form-link:hover {
  color: #999;
}
```